### PR TITLE
Add io.github.SamuelSchlemperSchlemuel.SingWriter

### PR DESCRIPTION
**Pull request by Samuel-Schlemper-Schlemuel**
This app is used for writing in SignWriting. It's like a text editor, but focused on a specific written language.

### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:**

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements#control-over-domain
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: io.github.SamuelSchlemperSchlemuel.SingWriter.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+    "id" : "io.github.SamuelSchlemperSchlemuel.SingWriter",
```
Comment (hfiguiere) [inline io.github.SamuelSchlemperSchlemuel.SingWriter.json:2]: app-id doesn't match github's

File: io.github.SamuelSchlemperSchlemuel.SingWriter.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "modules" : [
+        {
+            "name" : "SingWriter",
+            "builddir" : true,
+            "buildsystem" : "meson",
+            "sources" : [
+                {
+                    "type" : "git",
+                    "url" : "https://github.com/Samuel-Schlemper-Schlemuel/SingWriter",
+                    "branch": "main"
```
Comment (hfiguiere) [inline io.github.SamuelSchlemperSchlemuel.SingWriter.json:34]: NEVER a git branch. Tag and/or commit is required.

File: io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml
Comment (hfiguiere) [inline io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml:1]: this file should be upstream. I repeat from the closed PR.

File: io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  <p>Write in SignWriting (the write form of sign languages (Ex: ASL and Libras)) and convert the text to PDF</p>
+  <p xml:lang="pt_BR">Escreva em SignWriting (a forma escrita das línguas de sinais (Ex: ASL e Libras)) e converta o texto para PDF</p>
+  <p>The application uses the open font Noto Sans SignWriting. If your computer does not have this font, the application will not work properly. Install the font before using the application.</p>
+  <p xml:lang="pt_BR">O aplicativo usa a fonte aberta Noto Sans SignWriting. Se seu computador não tem essa fonte, o aplicativo não ira funcionar corretamente. Instale a fonte antes de usar o aplicativo.</p>
+</description>
+
+<launchable type="desktop-id">org.flatpak.gnome.desktop</launchable>
+
+<icon type="stock">io.github.SamuelSchlemperSchlemuel.SingWriter.svg</icon>
+<icon type="remote">https://raw.githubusercontent.com/Samuel-Schlemper-Schlemuel/SingWriter/f3bb637495e2d2ed384992dd5b1c187df62cfeed/data/icons/hicolor/scalable/apps/io.github.SamuelSchlemperSchlemuel.SingWriter.svg</icon>
```
Comment (hfiguiere) [inline io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml:27]:
```suggestion
```

suggestion still apply from the close PR

File: io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+<id>io.github.SamuelSchlemperSchlemuel.SingWriter</id>
+<metadata_license>CC0-1.0</metadata_license>
+<project_license>GPL-3.0-or-later</project_license>
+
+<name translate="no">SingWriter</name>
+<summary>Write in SignWriting</summary>
+<summary xml:lang="pt_BR">Escreva em SignWriting</summary>
+
+<developer id="io.github.SamuelSchlemperSchlemuel">
```
Comment (bbhtt) [inline io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml:13]:
```suggestion
<developer id="io.github.Samuel_Schlemper_Schlemuel">
```

File: io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+</developer>
+
+<description>
+  <p>Write in SignWriting (the write form of sign languages (Ex: ASL and Libras)) and convert the text to PDF</p>
+  <p xml:lang="pt_BR">Escreva em SignWriting (a forma escrita das línguas de sinais (Ex: ASL e Libras)) e converta o texto para PDF</p>
+  <p>The application uses the open font Noto Sans SignWriting. If your computer does not have this font, the application will not work properly. Install the font before using the application.</p>
+  <p xml:lang="pt_BR">O aplicativo usa a fonte aberta Noto Sans SignWriting. Se seu computador não tem essa fonte, o aplicativo não ira funcionar corretamente. Instale a fonte antes de usar o aplicativo.</p>
+</description>
+
+<launchable type="desktop-id">org.flatpak.gnome.desktop</launchable>
```
Comment (bbhtt) [inline io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml:24]: This is a copy paste mistake.

File: io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+    </description>
+  </release>
+</releases>
+
+<keywords>
+  <keyword>SignWriting</keyword>
+  <keyword xml:lang="pt_BR">Escrita de sinais</keyword>
+</keywords>
+
+<translation type="gettext">gnome-builder</translation>
```
Comment (bbhtt) [inline io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml:68]: Another copy paste mistake.

**Comment by bbhtt** [comment]
bot, build io.github.SamuelSchlemperSchlemuel.SingWriter

File: io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml
Comment (Samuel-Schlemper-Schlemuel) [reply io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml:1]: A doubt, how I can make this (if possible in graphical interface, but I don't have a problem in using the terminal)? I was searching how to make this, but I don't understand yet. The other requests I already maked.

File: io.github.SamuelSchlemperSchlemuel.SingWriter.json (hunk 1 above)
Comment (hfiguiere) [reply io.github.SamuelSchlemperSchlemuel.SingWriter.json:2]: this is the blocker.

https://docs.flathub.org/docs/for-app-authors/requirements#application-id

File: io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml
Comment (hfiguiere) [reply io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml:1]: I'll reiterate. This file should be managed where the source code of the app is, including being installed by the build system as to not be in the flathub repository. it is not flatpak specific.

File: io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml (hunk 4 above)
Comment (hfiguiere) [reply io.github.SamuelSchlemperSchlemuel.SingWriter.metainfo.xml:13]: this also apply to the application id, for the same reasons.

**Comment by bbhtt** [comment]
Why did you close this? The id change didn't require closing... now the review history is lost.

**Pull request closed without merging**